src/views/search/search-page.tsx

```diff
@@ -322,11 +333,7 @@ export function SearchPageBase({ siteTitle }: RootSearchPageProps) {
 	const numberOfPosts = showArticles ? data.totalPosts : 0;
 
 	return (
-		<main
-			className={style.fullPageContainer}
-			data-hide-sidebar={!query.searchQuery}
-		>
-			<h1 className={"visually-hidden"}>Search</h1>
+		<>
```

Comment:
Could the `useEffect` be avoided by making this a `<div>` to hold the data-hide-sidebar attribute instead? (and removing the styles from the`<main>` element in search.astro)

```suggestion
		<div
			className={style.fullPageContainer}
			data-hide-sidebar={!query.searchQuery}
		>
```

Reply:
Done.